src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCreateCollectorFileServiceImpl.java

```diff
@@ -43,19 +44,33 @@
     protected String collectorDirectoryPath;
 
     @Override
-    public String buildCollectorFile(ConcurStandardAccountingExtractFile saeFileContents) {
-        CollectorBatch collectorBatch = buildCollectorBatch(saeFileContents);
+    public String buildCollectorFile(ConcurStandardAccountingExtractFile saeFileContents,
+            ConcurStandardAccountingExtractBatchReportData reportData) {
+        if (saeFileContents == null) {
+            throw new IllegalArgumentException("saeFileContents cannot be null");
+        } else if (reportData == null) {
+            throw new IllegalArgumentException("reportData cannot be null");
+        }
+        

```

Comment:
I don't think it's strictly necessary for unchecked exceptions, but I'll check the Javadocs to make sure that at least the documentation mentions it.